server/test/test_anndata_adaptor.py

```diff
@@ -237,14 +237,12 @@ def test_compute_embedding(self):
                 self.data.compute_embedding("umap", filter)
             return
 
-        (schema, fbs) = self.data.compute_embedding("umap", filter)
+        schema = self.data.compute_embedding("umap", filter)
 
         self.assertIsInstance(schema["name"], str)
         name = schema["name"]
         self.assertEqual(schema["type"], "float32")
         self.assertEqual(schema["dims"], [f"{name}_0", f"{name}_1"])
 
-        emb = decode_fbs.decode_matrix_FBS(fbs)
-        self.assertEqual(emb["n_rows"], 100)
-        self.assertEqual(emb["n_cols"], 2)
-        self.assertEqual(emb["col_idx"], [f"{name}_0", f"{name}_1"])
+        emb = self.data.data.obsm[f"X_{name}"]
+        self.assertEqual(emb.shape, (2638, 2))
```

Comment:
Wouldn't you also want to assert that the re-embedding is only available for the 100 obs you subsetted and the rest are NaNs?

Reply:
good idea